keel-tagging-plugin/src/main/kotlin/com/netflix/spinnaker/keel/tagging/ResourceTagger.kt

```diff
@@ -195,8 +238,8 @@ class ResourceTagger(
       account
     }
 
-    val (region, resourceId) = if (suffix.contains(":")) {
-      suffix.split(":", limit = 2).toList()
+    val (region, resourceId) = if (toString().count { it == ':' } == 4) {
```

Comment:
This fixes a recently introduced bug where tags are improperly generated except when region is `*`